syntax = "proto3";

package asgt.type;

option go_package = "type";

import "asgt/type/data.proto";
import "asgt/type/target_value.proto";

message Sample {
  asgt.type.Data data = 1;
  repeated asgt.type.TargetValue target_values = 2;
}